Update share datasets (#225)
axiomhq/docs: reference/datasets.mdx

To vacuum fields, follow these steps:

1. Click <img src="/doc-assets/icons/settings.svg" className="inline-icon" alt="Settings icon" /> **Settings > Datasets**.
1. In the list, find the dataset where you want to vacuum fields, and then click <img src="/doc-assets/icons/vacuum.svg" className="inline-icon" alt="Vacuum fields icon" /> **Vacuum fields** on the right.
1. Select the checkbox, and then click **Vacuum**.

## Share datasets

You can share your datasets with other Axiom organizations. The receiving organization:

- can query the shared dataset.
- can create other Axiom resources that rely on query access such as dashboards and monitors.
- can’t ingest data into the shared dataset.
- can‘t modify the shared dataset.

> The receiving organization can’t generate ingest charges and is billed for the queries they run on the shared dataset. All query costs associated with a shared dataset accrues to the organization running the query.

Reviewer: ```suggestion
No ingest usage associated with the shared dataset accrues to the receiving organization. Query usage associated with the shared dataset accrues to the organization running the query.
```
A few suggestions baked in here:
1. Clarify that the comment around ingest charges is only relevant for the shared dataset
2. Remove the implication that running queries <-> being billed for queries. Typical usage will likely lead to no billing implications for the receiving organization, because usage will fall within quota
3. Use 'usage' rather than 'costs'/'charges', same reasoning as above

To share a dataset with another Axiom organization:

1. Ensure you have the necessary privileges to share datasets. By default, only users with the Owner role can share datasets.
1. Click <img src="/doc-assets/icons/settings.svg" className="inline-icon" alt="Settings icon" /> **Settings > Datasets**.
1. In the list, find the dataset that you want to share, and then click <img src="/doc-assets/icons/share-dataset.svg" className="inline-icon" alt="Share dataset icon" /> **Share dataset** on the right.
1. In the Sharing links section, click **+** to create a new sharing link.
1. Copy the URL and share it with the receiving user in the organization with which you want to share the dataset. For example, `https://app.axiom.co/s/dataset/{sharing-token}`.

> 1. Ask the receiving user to open the sharing link. When opening the link, the receiving user sees details about the dataset and your organization. They click **Add dataset** to confirm that they want to receive the shared dataset.

Reviewer: ```suggestion
1. Ask the receiving user to open the sharing link. When opening the link, the receiving user sees the name of the dataset and the email address of the Axiom user that created the sharing link. They click **Add dataset** to confirm that they want to receive the shared dataset.
```

We don't pass org info through (users can choose their org name)

### Delete sharing link

> Organizations can join the dataset with an active sharing link. To deactivate the sharing link, delete the sharing link. Deleting a sharing link means that organizations that don’t have access to the dataset can’t use the sharing link to join the dataset in the future. Deleting a sharing link doesn’t affect the access of organizations that already have access to the shared dataset.

Reviewer: Are you sure you want to use `join` here?

To delete a sharing link:

1. Click <img src="/doc-assets/icons/settings.svg" className="inline-icon" alt="Settings icon" /> **Settings > Datasets**.
1. In the list, find the dataset, and then click <img src="/doc-assets/icons/share-dataset.svg" className="inline-icon" alt="Share dataset icon" /> **Share dataset** on the right.
1. To the right of the sharing link, click <img src="/doc-assets/icons/delete.svg" className="inline-icon" alt="Delete icon" /> **Delete**.
1. Click **Delete sharing link**.

### Remove access to shared dataset

If your organization has previously shared a dataset with a receiving organization, and you want to remove the receiving organization’s access to the dataset, follow these steps:

> 1. Ensure you’re the owner of the dataset.

Reviewer: Not sure what's meant by this. Ensure you belong to the organization that owns the dataset? Ensure you have permissions to modify access to the shared dataset?
